core/tests/protractor/userManagement.js

```diff
@@ -24,16 +24,19 @@ var workflow = require('../protractor_utils/workflow.js');
 
 describe('Account creation', function() {
   it('should create users', function() {
-    users.createUser('ordinaryuser@example.com', 'ordinaryuser');
+    users.createUser(
+      'ordinaryuser@userManagement.com', 'ordinaryuser_userManagement');
```

Comment:
So should it be ordinaryUserUserManagement or ordinaryUserManagement ,
Do I have to  use camelcase in this particular case or in all of the usernames ?